src/controllers/words.js

```diff
@@ -43,8 +49,8 @@ export const searchWordUsingEnglish = async ({ query, searchWord, ...rest }) =>
   return sortDocsBy(searchWord, words, 'definitions[0]');
 };
 
-/* Gets words from MongoDB */
-export const getWords = async (req, res, next) => {
+/* Reuseable base controller function for getting words*/
+const getWordsBaseFunction = async (req, res, next, wordClass) =>{
```

Comment:
Let's change this function name to remove `Function` from the name. You can probably call it `getWordsFromDatabase` or `requestWords`, or something else.